koku/masu/database/sql/openshift/cost_model/distribute_platform_cost.sql

```diff
@@ -0,0 +1,178 @@
+    NULL as pod_request_memory_gigabyte_hours,
+    NULL as pod_effective_usage_memory_gigabyte_hours,
+    NULL as pod_limit_memory_gigabyte_hours,
+    max(node_capacity_cpu_cores) as node_capacity_cpu_cores,
+    max(node_capacity_cpu_core_hours) as node_capacity_cpu_core_hours,
+    max(node_capacity_memory_gigabytes) as node_capacity_memory_gigabytes,
+    max(node_capacity_memory_gigabyte_hours) as node_capacity_memory_gigabyte_hours,
+    max(cluster_capacity_cpu_core_hours) as cluster_capacity_cpu_core_hours,
+    max(cluster_capacity_memory_gigabyte_hours) as cluster_capacity_memory_gigabyte_hours,
+    NULL as persistentvolumeclaim,
+    NULL as persistentvolume,
+    NULL as storageclass,
+    NULL as volume_labels,
+    NULL as persistentvolumeclaim_capacity_gigabyte,
+    NULL as persistentvolumeclaim_capacity_gigabyte_months,
+    NULL as volume_request_storage_gigabyte_months,
+    NULL as persistentvolumeclaim_usage_gigabyte_months,
+    UUID '{{source_uuid | sqlsafe}}' as source_uuid,
+    'platform_distributed' as cost_model_rate_type,
+    CASE
+        WHEN {{distribution}} = 'cpu' AND (cost_category_id IS NULL OR max(cat.name) != 'Platform')
+            THEN sum(pod_effective_usage_cpu_core_hours) / max(udps.usage_cpu_sum) * max(pc.platform_cost)::decimal
+        WHEN {{distribution}} = 'memory' AND (cost_category_id IS NULL OR max(cat.name) != 'Platform')
+            THEN sum(pod_effective_usage_memory_gigabyte_hours) / max(udps.usage_memory_sum) * max(pc.platform_cost)::decimal
+        WHEN max(cat.name) = 'Platform'
+            THEN 0 - SUM(COALESCE(infrastructure_raw_cost, 0) +
+                COALESCE(infrastructure_markup_cost, 0)+
+                COALESCE(cost_model_cpu_cost, 0) +
+                COALESCE(cost_model_memory_cost, 0) +
+                COALESCE(cost_model_volume_cost, 0)
+            )
+    END AS distributed_cost,
+    max(cost_category_id) as cost_category_id
+FROM {{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary AS lids
+JOIN platform_cost as pc
+    ON pc.usage_start = lids.usage_start
+    AND pc.source_uuid = lids.source_uuid
+    AND pc.cluster_id = lids.cluster_id
+LEFT JOIN user_defined_project_sum as udps
+    ON udps.usage_start = lids.usage_start
+    AND udps.source_uuid = lids.source_uuid
+    AND udps.cluster_id = lids.cluster_id
+LEFT OUTER JOIN {{schema | sqlsafe}}.reporting_ocp_cost_category AS cat
+    ON lids.cost_category_id = cat.id
+WHERE lids.usage_start >= {{start_date}}::date
+    AND lids.usage_start <= {{end_date}}::date
+    AND report_period_id = {{report_period_id}}
+    AND lids.namespace IS NOT NULL
+    AND data_source = 'Pod'
+    AND node_capacity_cpu_core_hours IS NOT NULL
+    AND node_capacity_cpu_core_hours != 0
+    AND cluster_capacity_cpu_core_hours IS NOT NULL
+    AND cluster_capacity_cpu_core_hours != 0
+    AND lids.namespace != 'Worker unallocated'
+    AND lids.source_uuid = {{source_uuid}}
```

Comment:
Since report_period_id is in play, we can remove source uuid from the filter here, too.